### ISSUE TITLE
Builds fail to serve pages when they are moved after building

### ISSUE THREAD

**Issue by eweilow** [zeit/next.js#3703]
I'm deploying a project of mine to Heroku (which appear to be copying the project from a directory in which it is built before starting it). This causes issues with Next.js `5.0.0`, which appear to output absolute paths. This can be confirmed by going through the sources. 

This was not an issue with `4.2.3`.

- [x] I have searched the [issues](https://github.com/zeit/next.js/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
As the default behaviour (and I'm not sure that it can be controlled) on at least Heroku seem to be to build in one directory and run the build in another, I would expect a build to not contain absolute paths. 

## Current Behavior
I can see, for instance in `dist/bundles/pages/_document.js`, that absolute paths are being generated.
An example of an error (not in the test repository) is
```javascript
Error: Cannot find module '/tmp/build_<id>/eweilow-nextjs-deploy-test-<buildId>/node_modules/next/dist/pages/_document.js'
  at Function.Module._resolveFilename (module.js:555:15)
  at Function.Module._load (module.js:482:25)
  at Module.require (module.js:604:17)
  at require (internal/module.js:11:18)
  at Object.6 (/app/.next/dist/bundles/pages/_document.js:86:18)
  at __webpack_require__ (/app/.next/dist/bundles/pages/_document.js:23:31)
  at Object.5 (/app/.next/dist/bundles/pages/_document.js:78:18)
  at __webpack_require__ (/app/.next/dist/bundles/pages/_document.js:23:31)
  at /app/.next/dist/bundles/pages/_document.js:70:18
  at Object.<anonymous> (/app/.next/dist/bundles/pages/_document.js:73:10) code: 'MODULE_NOT_FOUND'
```

## Steps to Reproduce (for bugs)

1. Deploying https://github.com/eweilow/nextjs-deploy-test to Heroku with their default configuration fails.

## Your Environment

| Tech    | Version |
|---------|---------|
| next    | 5.0.0 |
| node    | 9.x |
| OS      | Ubuntu 16.04 on Heroku |
| browser | not relevant |
| etc     |         |

**Comment by timneutkens** [comment]
Duplicate of #3533
